Tests/StytchCoreTests/B2BMagicLinksTestCase.swift

```diff
@@ -195,6 +195,22 @@ extension MemberSession {
             expiresAt: refDate.advanced(by: 60 * 60 * 24),
             authenticationFactors: [],
             customClaims: nil,
+            roles: ["reader"],
```

Comment:
I feel like all these mocks should be moved out of the test case and into their own file. Thoughts?

Reply:
Yeah this is definitely something to do.